pkg/server/server.go

```diff
@@ -136,7 +130,7 @@ func (s *Server) ListObjects(ctx context.Context, req *openfgapb.ListObjectsRequ
 	return q.Execute(ctx, &openfgapb.ListObjectsRequest{
 		StoreId:              storeID,
 		ContextualTuples:     req.GetContextualTuples(),
-		AuthorizationModelId: modelID,
+		AuthorizationModelId: typesys.GetAuthorizationModelID(), // resolved model id
```

Comment:
the `q` object now has a reference to the `typesys` so why do we need to pass a field of `typesys` to `Execute()`?

Reply:
We don't, but since `Execute` accepts the `*openfgapb.ListObjectsRequest` this is the accurate way of passing the resolved authorization model id.

I didn't feel it was pertinent to refactor all of this in the scope of this PR.